src/lib/controller/index.ts

```diff
@@ -8,7 +8,7 @@ import https from "https";
 import { Capability } from "../capability";
 import { MutateResponse, AdmissionRequest, ValidateResponse } from "../k8s";
 import Log from "../logger";
-import { MetricsCollector } from "../metrics";
+import { metricsCollector, MetricsCollector } from "../metrics";
```

Comment:
```suggestion
import { metricsCollector } from "../metrics";
```

If we're moving to an externally-injected metrics collector, what are we still needing the `MetricsCollector` import for?

I see a single use of it down on [line 200](https://github.com/defenseunicorns/pepr/pull/983/files#diff-377ed79d843589a7aad90008c6f60d5276874a9b517d59ab6ff887675405b207R200) but is that right?  Everything else in the file is using a `this.#metricsCollector` but that still calls out to the global class..?

Reply:
It has a static method, [observeStart](https://github.com/defenseunicorns/pepr/blob/ad03b613c0abbc97d58afca4c069ad495f759712/src/lib/controller/index.ts#L200) that is used in the controller, didn't want to change "interfaces" and tests (maybe) if i could avoid it